test/parallel/test-fs-open-numeric-flags-posix.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+var common = require('../common');
+
+if (common.isWindows) {
+  console.log('1..0 # Skipped: open(2) is UNIX specific');
+  return;
+}
+
+var constants = require('constants');
+var assert = require('assert');
+var fs = require('fs');
+var path = require('path');
+
+common.refreshTmpDir();
+
+var openPath = path.join(common.tmpDir, 'file-should-not-exist');
+
+// A simple case: O_WRONLY without O_CREAT shall fail with ENOENT
+assert.throws(
+  function() {
+    fs.openSync(openPath, constants.O_WRONLY);
+  },
+  function(e) {
+    return e.code === 'ENOENT';
+  }
```

Comment:
This is fine but you can condense it to `err => e.code === 'ENOENT'` (arrow functions are allowed) or even just `/ENOENT/`.